test/lib/jdk/test/lib/cds/CDSArchiveUtils.java

```diff
@@ -494,6 +495,12 @@ public static File deleteBytesAtTheEnd(File orgFile, String newFileName) throws
         return dstFile;
     }
 
+    // used region size
+    public static long usedRegionSize(File archiveFile, int region) throws Exception {
+        long offset = spOffset + cdsFileMapRegionSize * region + spUsedOffset;
+        return readInt(archiveFile, offset, sizetSize);
+    }
+
     // used region size
     public static long usedRegionSizeAligned(File archiveFile, int region) throws Exception {
         long offset = spOffset + cdsFileMapRegionSize * region + spUsedOffset;
```

Comment:
Can you just call `usedRegionSize` like the following?
`long used = usedRegionSize(archiveFile, region);`